docs/sources/alerting/alerting-rules/templating-labels-annotations.md

```diff
@@ -1,5 +1,7 @@
 ---
-canonical: https://grafana.com/docs/grafana/latest/alerting/fundamentals/annotation-label/variables-label-annotation/
+aliases:
+  - ../fundamentals/annotation-label/variables-label-annotation/
```

Comment:
Would you mind adding a comment here that refers to the absolute path this relative alias redirects? It makes it easier to update these in the future. Like https://grafana.com/docs/writers-toolkit/write/front-matter/#aliases:~:text=%2D%20./original%2Durl/%20%23%20/docs/grafana/%3CGRAFANA_VERSION%3E/original%2Durl/

```suggestion
  - ../fundamentals/annotation-label/variables-label-annotation/ # /docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/annotation-label/variables-label-annotation/
```

Reply:
addressed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344